tests/lib/sources/test_base.py

```
@@ -0,0 +1,19 @@
+import pytest
+from src.lib.sources.base import ComicSource
+
+class DummySource(ComicSource):
+    def fetch_page(self, url):
```

Ruff (0.13.1)

Unused method argument: `url`

(ARG002)